src/chapters/spring-model-validation/exercises.rst

```diff
@@ -0,0 +1,28 @@
+Exercises: Model Validation
+===========================
+
+Let’s practice adding more fields onto our event objects and 
+validating them. Create a new branch from TODO: STARTING CODE BRANCH. For each of the items below,
```

Comment:
An additional clarifying sentence could help, before "For each of the items..." When initially reading this, it took me a moment to contextualize this info properly. Something like: "Below, we describe some new fields for you to add to the ``Event`` class. For each field, consider the following factors:"